includes/collection/class-replies.php

```diff
@@ -0,0 +1,175 @@
+<?php
+namespace Activitypub\Collection;
+
+use WP_Post;
+use WP_Comment;
+use WP_Error;
+
+use function Activitypub\is_local_comment;
+use function Activitypub\get_rest_url_by_path;
+
+/**
+ * Class containing code for getting replies Collections and CollectionPages of posts and comments.
+ */
+class Replies {
```

Comment:
I think we should merge that class with the interactions collection. I have no emotions about which one to use, so we can wait until this is merged and migrate the interactions class afterwards. Just wanted to mention it.

I think maybe `Replies` is really the better choice!

Reply:
I prefer smaller files and think having this under collection/ is a good place.

Reply:
I agree that this should be a collection and I will think about the "small" files :)